### ISSUE TITLE
Document uses artificial attribute data-lang instead of the built-in lang attribute

### ISSUE THREAD

**Issue by aphillips** [w3c/clreq#71]
A colleague (Robert Chu) pointed out to me:

> The Chinese text in the document aren’t properly tagged - http://w3c.github.io/clreq/ 
> 
> Example:
> 
> `<p data-lang="zh">中文分為繁體字與簡體字，兩者除文字字形、筆畫多寡等差異外，各項排版規則也有所區別。</p>`
> 
> Not sure why ‘data-lang’ was used all over the place rather than ‘lang’.

There doesn't appear to be a reason not to use the `lang` attribute. The language selection script and styling can work on that attribute just as well (even better, in the case of `:lang`). Failing that, even if `data-lang` is kept, the text should still be marked up with `lang` so that browsers that implement language-based font selection and other features can use it.

**Comment by r12a** [comment]
it is true that we should mark up the content with the lang attribute (and i plan to do that at some point, but not before FPWD), but data-lang was chosen for the bilingual management on purpose, since the use of the lang attribute may not always coincide with the _structural_ usage for which data-lang is designed.  

Both lang and data-lang mean different things and are used in different ways.  The lang attribute means just "the text in this element is in the declared language".  The data-lang attribute means "the text in this attribute is part of a pair of elements that make up a unit of content".  

This is most clearly seen where someone creates new content in chinese, but is unable to create the corresponding english text.  The guidelines (which i have prepared and will upload tomorrow now) say that both the data-lang=zh and the data-lang=en fields must be created at the same time, and if there is no english available, use the chinese twice.  This ensures that all content is available when one of the language filters (top right) is applied, and the structure is always properly balanced.  It also helps quickly identify items that are in need of translation – and will be a significant improvement over the process we've been using so far.  If we used the lang attribute the filters wouldn't work in the same way.  

Additionally, there are likely (as the text and inline markup is developed - it's still at an early stage, esp. for the Chinese), that there will be additional inline ranges to which the lang attribute is applied to mark language differences. These are irrelevant to the structural view of the document, and would make the filtering process much more complicated, and perhaps less successful, if we relied only on lang attributes.

Bottom line, lang attribute and data-lang attribute are doing different things, although there is likely to be some overlap, and they actually are properly tagged, although we do still need to add lang attributes for better font control.  (I'd like to explore the possibility of doing some of that via javascript, btw, to save work for the editors.)  For the time being, we have mixed SC and TC, so font issues are a secondary concern just now.

**Comment by aphillips** [comment]
That make sense (plus: the English is likely enough to quote runs of Chinese, which in turn wants `lang` markup). But you'll want `lang` in the doc--having just `zh` is even helpful.
